MAUI/Shimmer/Customization.md

```diff
@@ -0,0 +1,239 @@
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+SfShimmer Shimmer = new SfShimmer()
+{
+      Type = ShimmerType.Article,
+      Fill = Colors.Pink,
+      WaveColor = Color.FromRgba("#9A4EAE"),
+      WaveWidth = 100,
+      Content = new Label
+      {
+         Text = "Content is loaded!"              
+      }
+};
+
+this.Content = Shimmer;
+
+{% endhighlight %}
+{% endtabs %}
+
+## WaveDirection
+
+The direction of wave in `SfShimmer` view can be customized by using the `WaveDirection` property. By default, wave direction is rendered by `Default` wave. The following wave directions are available in SfShimmer:
+
+* `Default` - Default enum of the animation direction allows the animation from left top to right bottom.
+* `LeftToRight` - LeftToRight enum of the animation direction allows the animation from left to right.
+* `RightToLeft` - RightToLeft enum of the animation direction allows the animation from right to left.
+* `TopToBottom` - TopToBottom enum of the animation direction allows the animation from top to bottom.
+* `BottomToTop` - BottomToTop enum of the animation direction allows the animation from bottom to top.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<shimmer:SfShimmer x:Name="Shimmer" VerticalOptions="Fill" Type="Article" WaveDirection="TopToBottom">
+   <shimmer:SfShimmer.Content>
+      <StackLayout>
+         <Label 
+            Text="Content is loaded!!">
+         </Label>
+      </StackLayout>
+   </shimmer:SfShimmer.Content>
+</shimmer:SfShimmer>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+SfShimmer Shimmer = new SfShimmer()
+{
+      Type = ShimmerType.Article,
+      WaveDirection = ShimmerWaveDirection.TopToBottom,
+      Content = new Label
+      {
+         Text = "Content is loaded!"              
+      }
+};
+
+this.Content = Shimmer;
+
+{% endhighlight %}
+{% endtabs %}
+
+## RepeatCount
+
+The `RepeatCount` of the shimmer refers to the number of times that a shimmer cycle repeats within a given time period of wave in `SfShimmer` view. The default value of `RepeatCount` is `1`.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<shimmer:SfShimmer x:Name="Shimmer" VerticalOptions="Fill" Type="Article" RepeatCount="4">
+   <shimmer:SfShimmer.Content>
+      <StackLayout>
+         <Label 
+            Text="Content is loaded!!">
+         </Label>
+      </StackLayout>
+   </shimmer:SfShimmer.Content>
+</shimmer:SfShimmer>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+SfShimmer Shimmer = new SfShimmer()
+{
+      Type = ShimmerType.Article,
+      RepeatCount = 4,
+      Content = new Label
+      {
+         Text = "Content is loaded!"              
+      }
+};
+
+this.Content = Shimmer;
+
+{% endhighlight %}
+{% endtabs %}
+
+## AnimationDuration
+
+The duration of the animation in `SfShimmer` view can be customized by using the `AnimationDuration` property. The default value of `AnimationDuration` is 1000 ms.
```

Comment:
The duration of the animation in the `SfShimmer` view can be customized using the `AnimationDuration` property. The default value of `AnimationDuration` is 1000 ms.

Reply:
Content modified